addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -99,6 +99,7 @@ def _prepare_invoice_values(self, order, name, amount, so_line):
                 'analytic_tag_ids': [(6, 0, so_line.analytic_tag_ids.ids)],
                 'analytic_account_id': order.analytic_account_id.id if not so_line.display_type and order.analytic_account_id.id else False,
             })],
+            'invoice_incoterm_id': order.incoterm.id,
```

Comment:
sale.order().incoterm seems to be added by module `sale_stock`:

https://github.com/odoo/odoo/blob/adf21721c4c031d5f2002a0f816764a2df20006f/addons/sale_stock/models/sale_order.py#L26

thus we can't make a reference to it in a module that is not installed when sale_stock is not installed. I think the fix should rather in addons/sale_stock/models/ and be an override similar to this:

https://github.com/odoo/odoo/blob/15.0/addons/l10n_in_sale/wizard/sale_make_invoice_advance.py